lib/plugins/create/templates/aws-nodejs-docker/schema.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this file specific to Docker, why is it needed?

Reply:
I've added it as a schema for `http` event, but if you feel like it's too much boilerplate for a simple template I'm fine with removing it :+1:

Reply:
I think I would remove it, at least I guess we do not practise adding this in other templates (?)

Reply:
I took inspiration from `aws-nodejs-typescript` template, do you suggest to remove schema only or the whole `http` event definition?

Reply:
Maybe I miss something, but I don't see this file in `aws-nodejs-typescript` template -> https://github.com/serverless/serverless/tree/b753641b072485d4764e891b5e90242776bec724/lib/plugins/create/templates/aws-nodejs-typescript

Also I think base from which we derive should rather be `aws-nodejs` template -> https://github.com/serverless/serverless/tree/b753641b072485d4764e891b5e90242776bec724/lib/plugins/create/templates/aws-nodejs

Reply:
`aws-nodejs-typescript` is structured differently and the schema is defined here: https://github.com/serverless/serverless/blob/b753641b072485d4764e891b5e90242776bec724/lib/plugins/create/templates/aws-nodejs-typescript/src/functions/hello/schema.ts as it composes `serverless.ts` file instead of using `serverless.yml`. But I think you're right, I'll just remove the `http` event altogether and leave it to be super simple, similar to `aws-nodejs` template :+1:

Reply:
Yes, it looks TS specific (or at least influenced by it). I wouldn't mix this into non TS template.

Otherwise if you feel some parts in TS template are valuable to have in our base template, we may discuss and extend it (having that we can also add it here)

Reply:
I don't think it's TS specific, as it's just a standard API Gateway request schema, the only different is in how it's defined in case of that particular template. The `aws-nodejs-typescript` template is more detailed, with additional libraries and so on - it might be worth considering to have such improvements in different templates, but I believe `examples` can serve a similar purpose. For now, I'll stick to a simpler template based on `aws-nodejs` :+1:

Reply:
Thanks @pgrzesik for clarifications, great feedback.